.github/workflows/main.yaml

```diff
@@ -30,7 +30,7 @@ jobs:
         check-latest: true
         go-version: ${{ env.GO_VERSION }}
     - name: golangci-lint
-      uses: golangci/golangci-lint-action@55c2c1448f86e01eaae002a5a3a9624417608d84 # v6.5.2
+      uses: golangci/golangci-lint-action@1481404843c368bc19ca9406f87d6e0fc97bdcfd # v7.0.0
```

Comment:
This one seems like it has some breaking changes. 

Build (lint) is failing:

<img width="1137" alt="image" src="https://github.com/user-attachments/assets/cd5228e2-cea1-43cc-bfab-d823fc2f6db8" />